common/flatpak-context.c

```diff
@@ -87,6 +87,7 @@ const char *flatpak_context_special_filesystems[] = {
   "host",
   "host-etc",
   "host-os",
+  "host-reset",
```

Comment:
I want to check what happens when we literally pass `--filesystem=host-reset` or `--filesystem=host-reset:rw` or `--nofilesystem=host-reset` on the command-line.

For best clarity, I think the answer should probably be that `--filesystem=host-reset` is an error, so that `--nofilesystem=host-reset` is the only form allowed.

Reply:
It now fails with `Filesystem token "host-reset" is only applicable for --nofilesystem`.